Specs/Scene/ExpressionSpec.js

```diff
@@ -2143,6 +2292,19 @@ defineSuite([
         }).toThrowDeveloperError();
     });
 
+    it('throws if min function takes mismatching types', function() {
```

Comment:
Changed min -> max in the correct location below. I believe it should be min here